client/src/components/pages/LivePosts/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import PostButton from 'components/utils/PostButton'
+import Post from 'components/utils/Post';
+import './style.css'
+
+class LivePosts extends Component {
+  state = {
+    livePosts: [],
+    error: '',
+  }
+
+  componentDidMount() {
+    //Mock Data for Testing the component//
+    const livePosts = [
+      { title: 'post 1', id: 1 },
+      { title: 'post 2', id: 2 },
+      { title: 'post 3', id: 3 },
+      { title: 'post 4', id: 4 },
+      { title: 'post 5', id: 5 },
+      { title: 'post 6', id: 6 },
+    ];
+    this.setState({ livePosts });
+  }
+
+  handleDelete = (id) => {
+    const { livePosts } = this.state;
+    //Testing Delete with Mock Data//
+    const deletedPost = livePosts.findIndex(post => post.id === Number(id));
+    this.setState((prevState) => {
+      prevState.livePosts.splice(deletedPost, 1);
+      return ({ livePosts: prevState.livePosts })
+    });
+  }
+
+  render() {
+    const { error, livePosts } = this.state;
+    return (
+      <section className='live-posts'>
+        {
+          error ?
+            <span className='live-posts-error'>{error}</span>
+            : ''
+        }
+        <PostButton className='post-type--bold' postType='Live Posts' />
+        {
+          livePosts.length ?
```

Comment:
@IsraaSulaiman If you don't want to have to include a `: ''` at the end of this block you can change to this:
```
{
  livePosts.length &&
    <div className='live-posts--container'> 
    .....
    </div>
}
```

Reply:
@tomduggan85 If I did as you suggested, `0` which is the length of the array when I delete all posts will be rendered in the page

Reply:
Ah whoops, I think I should’ve said “!!livePosts.length && ...”. That should fix the 0.

Reply:
.. or “livePosts.length > 0 && ...” 🤷‍♂️
Either one will convert the expression to a bool, so react will know to not render anything in that case.